packages/ppx_update/ppx_update.0.81/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "PPX library to optimize record updates"
+description:
+  "ppx_update optimizes away record updates when new record contents match old ones"
+maintainer: ["yotambarnoy@gmail.com"]
+authors: ["Yotam Barnoy"]
+license: "MIT"
+tags: ["ppx"]
+homepage: "https://github.com/bluddy/ppx_update"
+doc: "https://url/to/documentation"
```

Comment:
This field is optional, if there is no documentation page this should probably be removed
```suggestion
```

Reply:
done